boards/emonpi.json

```diff
@@ -0,0 +1,22 @@
+{
+  "build": {
+    "core": "arduino",
+    "extra_flags": "-DARDUINO_ARCH_AVR -DAVR_RASPDUINO",
+    "f_cpu": "16000000L",
+    "mcu": "atmega328p",
+    "variant": "standard"
+  },
+  "frameworks": [
+    "arduino"
+  ],
+  "name": "BitWizard Raspduino",
```

Comment:
Wops, sorry this was added in a later commit 20f7e8ad76ddff23d7da0557b78629f068fdb3f2